### ISSUE TITLE
Shut down BC signalling endpoint

### ISSUE THREAD

**Issue by peterbe** [mdn/kuma#7721]
**Summary**
There is no way to submit BC signaling any more. Can we remove all its functionality but keep the data (in [the admin](https://developer.mozilla.org/admin/wiki/bcsignal/)) until we decide to scrap that too. 

**Rationale**
There's no way to populate more of them since we rewrote how the BCD table works. 

**Proposal**
Delete the Django endpoint view and all its tests

**Additional context**
Postpone what to do with the data currently stored in MySQL for another day.

**Comment by peterbe** [comment]
Thoughts on this @Elchi3 @ddbeck @escattone ?

**Comment by Elchi3** [comment]
It was nice to look at compat table user feedback frequently to improve the data and the table rendering but access to that data was limited to Mozilla employees so I think it is of very limited use these days. Some evaluation I did is public here: https://github.com/mdn/sprints/issues/2289. The rest will remain with Mozilla Corp.

In Yari, this was replaced with a link to file public issues on BCD which makes this feedback public by default. This has its own problems (https://github.com/mdn/browser-compat-data/issues/8017) but I think it works for me. So, yeah, I would say shut it down.

Thanks again to the lovely Open Innovation people (Hermina Condei, Rosana Ardila, John Giannelos, Matt Croud) who made this possible in 2019.
